### ISSUE TITLE
FPF report exported doesn't show suppressed results

### ISSUE THREAD

**Issue by peterkline** [DependencyTrack/dependency-track#2790]
### Current Behavior

We are hitting the API endpoint: `/v1/finding/project/{uuid}/export` to get a report but the results that are generated do not show suppressed results.  I would suspect that since the json has an entry for the component for the analysis that has `isSuppressed` that it would still show up in the export but just have `"isSuppressed": true`

### Steps to Reproduce

1. Take a component and suppress it
2. Get an FPF report by hitting the /v1/finding/project/{uuid}/export api endpoint
3. View results and see that suppressed components don't show up in findings

### Expected Behavior

Expected Behavior is that exported FPF would show suppressed components with JSON like the following:
```json
      "analysis": {
        "isSuppressed": true,
        "state": "RESOLVED"
      },
```

### Dependency-Track Version

4.8.2

### Dependency-Track Distribution

Container Image

### Database Server

PostgreSQL

### Database Server Version

14

### Browser

Google Chrome

### Checklist

- [X] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [X] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this defect was already reported

**Comment by nscuro** [comment]
We could theoretically add a `suppressed` query parameter to this endpoint, which defaults to `false`. The important part here is to not break existing behavior, as existing integrations rely on exports *not* containing suppressed findings.

In the meantime, is using the `/api/v1/finding/project/{uuid}` an option for you? There you can already use `suppressed=true` to also get supressed findings.

**Comment by peterkline** [comment]
Another issue with not using the `/export` endpoint to get the FPF, the attributionDate is a different format:

Using `/api/v1/finding/project/{uuid}`
```json
    "attribution": {
      "attributedOn": 1684800840169
    },
```
Using `/api/v1/finding/project/{uuid}/export`
```json
      "attribution": {
        "attributedOn": "2023-05-17 18:34:47.393",
      },
```
So I would definitely like to see the option to add a `suppressed` query parameter for the `export` endpoint

**Comment by lsomaini** [comment]
Hi,
what is the time fomat of the attributedOn in `/api/v1/finding/project/{uuid}`?

**Comment by peterkline** [comment]
> Hi, what is the time fomat of the attributedOn in `/api/v1/finding/project/{uuid}`? `"attribution": { "attributedOn": 1684800840169 },`

This appears to be epoch time

**Comment by peterkline** [comment]
Yes I took two random components within the project so the datestamp was skewed
